org.OpenT2T.Onboarding.VeraHub/node/hub.js

```diff
@@ -0,0 +1,243 @@
+'use strict'; 
+
+var https = require('https');
+var cryptojs_sha1 = require('crypto-js');
+var q = require('q');
+
+var internal = 
+{
+    // global vera constants
+    veraHashBase : 'oZ7QE6LcLJp6fiWzdqZc',
```

Comment:
Could you please expand on what this hash corresponds to ? (just for documentation purpose)